sympy/functions/special/gamma_functions.py

```diff
@@ -116,8 +116,7 @@ def eval(cls, arg):
             elif intlike(arg):
                 if arg.is_positive:
                     return factorial(arg - 1)
-                else:
-                    return S.ComplexInfinity
+                return S.ComplexInfinity
```

Comment:
unnecessary change

Reply:
Due to this comment: https://github.com/sympy/sympy/pull/16843#discussion_r289607306

Reply:
The code is equivalent, but please don't modify working code.

Reply:
I actually prefer having `else:` even after `return`, I believe the code is a bit more readable. But that's a minor issue.

Reply:
Ok! I will remove the unnecessary changes.